src_py/apiServer/apiServer.py

```diff
@@ -8,12 +8,24 @@
 
 class ApiServer():
     def __init__(self): 
-
         mainServerIP = globe.components.mainServerIp
         mainServerPort = globe.components.mainServerPort
         self.mainServerAddress = 'http://' + mainServerIP + ':' + mainServerPort
+
+        # Send the content of jsonPath to each devices:
+        print("Sending jsonPath to devices...\n")
```

Comment:
Why does it send the json path to devices and not the json itself? 
Please explain me this line in email or whatsapp 
@dordor7